client/pool.c

```diff
@@ -508,6 +523,8 @@ static void register_new_miner(connection_list_element *connection, int index)
 		conn_data->miner = &new_miner->miner_data;
 		conn_data->state = ACTIVE_CONNECTION;
 	}
+
+	return 1;
```

Comment:
Usually use 0 for success, 1 for failure. Maybe keep the same rule it's better.

Reply:
My logic was 1 - success, 0 - failure. And the check for result of function looks like:
if(!function())   //if(!success)
{.....}